single-host-httpclient/src/test/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/SingleHostHttpClientBuilderTest.java

```diff
@@ -22,16 +22,21 @@
 import static org.junit.jupiter.api.Assertions.assertSame;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-class SingleHostHttpClientProviderTest {
+class SingleHostHttpClientBuilderTest {
+    static {
+        // Force properties
+        System.setProperty("jdk.internal.httpclient.disableHostnameVerification", Boolean.TRUE.toString());
```

Comment:
It is set here and in the surfire plugin?

Reply:
Yes so that you don't have to set it in IDE